docs/sources/release-notes.md

```diff
@@ -89,4 +45,4 @@ impact on users. This list will be updated as issues are resolved.
 An idiosyncrasy in AUFS prevents permissions from propagating predictably
 between upper and lower layers. This can cause issues with accessing private
 keys, database instances, etc. For complete information and workarounds see
-[Github Issue 783](https://github.com/docker/docker/issues/783).
\ No newline at end of file
+[Github Issue 783](https://github.com/docker/docker/issues/783).
```

Comment:
Let's also add the Safari/Hub issue while we're in here:

** Docker Hub incompatible with Safari 8**

The Docker Hub has multiple issues running on Safari 8, the default browser for OS X 10.10 (Yosemite). Users should access the hub using a different browser.